web/helpers/product-creator.js

```diff
@@ -83,17 +83,27 @@ export default async function productCreator(session, count = DEFAULT_PRODUCTS_C
   const client = new Shopify.Clients.Graphql(session.shop, session.accessToken);
 
   for (let i = 0; i < count; i++) {
-    await client.query({
-      data: {
-        query: CREATE_PRODUCTS_MUTATION,
-        variables: {
-          input: {
-            title: `${randomTitle()}`,
-            variants: [{ price: randomPrice() }],
+    try {
```

Comment:
another nit, shouldn't the try/catch be around the for loop as well? just to totally bail out in case something went wrong?